### PR TITLE
[Bulk] Validation: Fixed missing descriptions 5/24

### PR DESCRIPTION
**Pull request by reemsabaw1**
This PR fixes 30 missing descriptions.

### CODE REVIEW

File: iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-clipprogressupdate-event-microsoft-web-media-smoothstreaming.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: SmoothStreamingMediaElement.ClipProgressUpdate Event (Microsoft.Web.Media.SmoothStreaming)
+description: ClipProgressUpdate Event occurs when media reaches a marker
```
Comment (AqMoh) [inline iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-clipprogressupdate-event-microsoft-web-media-smoothstreaming.md:3]: Add period to end of sentence.

File: iis/extensions/transform-manager/utilities-cloneelement-method-microsoft-web-media-transformmanager.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Utilities.CloneElement Method  (Microsoft.Web.Media.TransformManager)
```
Comment (AqMoh) [inline iis/extensions/transform-manager/utilities-cloneelement-method-microsoft-web-media-transformmanager.md:2]: Remove extra period between Method and (Microsoft...)

File: iis/web-development-reference/native-code-api-reference/iworkerprocessframework-getcustominterface-method.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "IWorkerProcessFramework::GetCustomInterface Method"
+description: IWorkerProcessFramework::GetCustomInterface Method retrieves a custom interface from the worker process framework
```
Comment (AqMoh) [inline iis/web-development-reference/native-code-api-reference/iworkerprocessframework-getcustominterface-method.md:3]: Add a period to end of setence.

File: iis/extensions/smooth-streaming-client/clipplaybackeventargs-methods-microsoft-web-media-smoothstreaming_1.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: ClipPlaybackEventArgs Methods (Microsoft.Web.Media.SmoothStreaming)
+description: The ClipPlaybackEventArgs type exposes the following members.
```
Comment (AqMoh) [inline iis/extensions/smooth-streaming-client/clipplaybackeventargs-methods-microsoft-web-media-smoothstreaming_1.md:3]: Duplicate description! Please remove one.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**